### PR TITLE
Add support for block comment rollback commands on SQL changesets

### PR DESCRIPTION
**Pull request by krishnaenugandula**
* [ ] Bug fix (non-breaking change which fixes an issue.)
* [x] Enhancement/New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Add support for block comment rollback commands on SQL changesets as in thefollowing example:

```
				/* liquibase rollback
				 delete from table2;
				 drop table table2;
				*/
```

Currently the only way to do a rollback like this one is by repeating the rollback comment tag:

```
-- rollback delete from table2;
-- rollback drop table table2;
```

## Actual Behavior
Block comments are not supported.

## Additional Context
It's a new  functionalitty thus nothing should be broken by it.

## Things to remember
**Documentation needs to be updated to include this new rollback option.**

## Fast Track PR Acceptance Checklist:

* [x] Build is successful and all new and existing tests pass
* [x] Added [Unit Test(s)](https://liquibase.jira.com/wiki/spaces/LB/pages/1274937609/How+to+Write+Liquibase+Core+Unit+Tests)
* [ ] Added [Integration Test(s)](https://liquibase.jira.com/wiki/spaces/LB/pages/1276608569/How+to+Write+Liquibase+Core+Integration+Tests)
* [ ] Documentation Updated

### CODE REVIEW

**Comment by krishnaenugandula** [comment]
Have added the multi line comment type rollback support.
This is the most annoying reason one has to hear while adapting liquibase.
Hence have added support. This make it complete adaptable.

**Comment by molivasdat** [comment]
Thanks @krishnaenugandula  for creating this enhancement request.
Here’s what happens next:

A member of the Liquibase team will take a look at your contribution and may suggest
changes, additional tests, or provide other feedback. The PR will be prioritized
according to our development and testing capacity.

We’ll let you know when it’s ready to move to the next step or if any changes are needed.

**Comment by filipelautert** [comment]
Closed #339 and #695 in behalf of this one.

@krishnaenugandula thanks foir the contribution! Do you think you would be able to fix the conflicts that this PR has right now?
Also, maybe "--multiline-rollback" can be replaced to something more on liquibase standards, like "liquibase rollback" or something like that? What do you think?

**Review by filipelautert — APPROVED** [review]
Code looks good and tests were added. A documentation PR needs to be created to update documentation.

**Comment by filipelautert** [comment]
Once PR is fully approved I can write the documentation changes to reflect it.

**Review by suryaaki2 — APPROVED** [review]

File: liquibase-core/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy [hunk 1]
```diff
@@ -792,6 +791,109 @@ not ignoreLines here
...
+);
+
+/* liquibase rollback
+ delete from table2;
+ drop table table2;
+*/
+
+--ChangeSet nvoxland:3
+select (*) from table3;
+--rollback not required
```
Comment (XDelphiGrl) [inline liquibase-core/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy:847]: Should this use the new `empty` keyword?

**Review by XDelphiGrl — CHANGES_REQUESTED** [review]
This PR extends Liquibase handling of `rollback` statements in formatted SQL changelogs to parse multiple statements within a comment block. An example of a comment block rollback with multiple statements is:
```
/* liquibase rollback
 delete from table1;
 drop table table1;
*/
```

* Automated functional and test harness executions passing.
* New unit tests added to validate multiple variations on multiline rollback statements.
* No additional testing required.

The branch is out of date with master. When that is updated and the tests passing, I'll approve the PR.

File: liquibase-core/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy (hunk 1 above)
Comment (filipelautert) [reply liquibase-core/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy:847]: it should, good catch!

**Comment by filipelautert** [comment]
@XDelphiGrl branch updated and tests reviewed.

File: liquibase-core/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy (hunk 1 above)
Comment (XDelphiGrl) [reply liquibase-core/src/test/groovy/liquibase/parser/core/formattedsql/FormattedSqlChangeLogParserTest.groovy]: Thanks for updating the test, @filipelautert !

**Review by XDelphiGrl — APPROVED** [review]
APPROVED

See [https://github.com/liquibase/liquibase/pull/1399#pullrequestreview-1199538032](https://github.com/liquibase/liquibase/pull/1399#pullrequestreview-1199538032) for details.

**Comment by emesar** [comment]
This functionality doesn't appear in any official Liquibase documentation, but appears to work from my testing - can/should this be added to the docs?

**Comment by adrian-velonis1** [comment]
@emesar Thanks for the heads up. We've created an internal ticket to document this functionality.

[DOC-INTERNAL] https://datical.atlassian.net/browse/PD-4463

**Pull request merged**